### DISCUSSION TITLE
Twitter Authentication - "Could not authenticate you" error

### DISCUSSION THREAD

**Discussion by steliosrammos** [nextauthjs/next-auth#2454] [Help]
### Description 🐜

When attempting to sign in with Twitter, I get the "Try signing in with a different account." error. The console does not output any error and the logs output a "get_authorization_url_error" and "signin_oauth_error" error both saying "Could not authenticate you." (see screenshots below).

### Is this a bug in your own project?

No

### How to reproduce ☕️

I used the `next-auth-example` project to try out NextAuth and followed the exact instructions in the documentation.

I have configured the Twitter App with the following callback URL: http://localhost:3000/api/auth/callback/twitter

In the .env.local, I set up the TWITTER_ID with the consumer key, the TWITTER_SECRET with the consumer secret key provided by the Twitter App and the NEXTAUTH_URL to http://localhost:3000.

I made no other change to the project.

### Screenshots / Logs 📽

<img width="525" alt="Screenshot 2021-07-29 at 20 38 41" src="https://user-images.githubusercontent.com/24457157/127547710-835b4305-92f7-4027-91bb-7275768c1c96.png">

<img width="333" alt="Screenshot 2021-07-29 at 20 39 03" src="https://user-images.githubusercontent.com/24457157/127547748-2e28092d-7622-4085-b4be-e26f5f92b343.png">

### Environment 🖥

  System:
    OS: macOS 12.0
    CPU: (12) x64 Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
    Memory: 754.36 MB / 16.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.5.0 - ~/.nvm/versions/node/v16.5.0/bin/node
    npm: 7.19.1 - ~/.nvm/versions/node/v16.5.0/bin/npm
  Browsers:
    Chrome: 92.0.4515.107
    Safari: 15.0
  npmPackages:
    next: ^11.0.0 => 11.0.1 
    next-auth: latest => 3.27.3 
    react: ^17.0.2 => 17.0.2 

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
There must be a configuration error somewhere.

At https://next-auth-example.vercel.app/ you can check that the Twitter login works as expected. It uses the `next-auth-example` repository code also. You must provide a proper reproduction if you need further help.

I would double check if the environment variable names are matching.

**Reply by steliosrammos** [reply]
Hey sorry if this is a misconfiguration. I have checked back on the environment variables as you point out and I just don't see it... 

I added below screenshots of all the relevant config lines, including Twitter key and secret that I regenerated to double check. Perhaps someone checking this will see what's wrong. 

<img width="566" alt="Screenshot 2021-07-30 at 23 03 45" src="https://user-images.githubusercontent.com/24457157/127712235-c11512c8-f460-4386-8b69-f5b7c73658e9.png">
<img width="602" alt="Screenshot 2021-07-30 at 23 02 50" src="https://user-images.githubusercontent.com/24457157/127712241-4f8b1a2a-8ec1-40f0-977c-6d16c23f25fc.png">
<img width="578" alt="Screenshot 2021-07-30 at 23 06 30" src="https://user-images.githubusercontent.com/24457157/127712247-ee64447a-9639-4e7c-b0be-678cf5710ff2.png">
<img width="399" alt="Screenshot 2021-07-30 at 23 01 12" src="https://user-images.githubusercontent.com/24457157/127712253-48f9f10d-72ab-4224-b7a6-fa07788d0ce7.png">
<img width="664" alt="Screenshot 2021-07-30 at 23 07 56" src="https://user-images.githubusercontent.com/24457157/127712257-b648ac35-6bf1-4ad9-9c6d-208950978028.png">

**Reply by mikhail-karan** [reply]
Twitter seems to no longer work in this example: https://next-auth-example.vercel.app/api/auth/signin?error=OAuthSignin
